### DISCUSSION TITLE
能用 Zeabur 进行部署吗？

### DISCUSSION THREAD

**Discussion by UangSC** [Qexo/Qexo#379] [Q&A]
因为我使用 Vercel 部署时候，打开部署项目的站点就必须开 VPN 才能正常使用
今天尝试了下使用 Zeabur 进行部署，但是打开页面后 Qexo error 500 页面。
所以不太明白是因为我配置出现问题，还是 Qexo 不适配 Zeabur 的部署？
抱歉，我不太了解这些技术，所以可能有些疑问有些问题...

**Comment by Fgaoxing** [comment]
你需要配置一个域名，因为Vercel给予的域名无法访问



在 2023-05-11 17:48:32，"UangSC" ***@***.***> 写道：

因为我使用 Vercel 部署时候，打开部署项目的站点就必须开 VPN 才能正常使用
今天尝试了下使用 Zeabur 进行部署，但是打开页面后 Qexo error 500 页面。
所以不太明白是因为我配置出现问题，还是 Qexo 不适配 Zeabur 的部署？
抱歉，我不太了解这些技术，所以可能有些疑问有些问题...

**Reply by UangSC** [reply]
了解了，感谢~

**Comment by am-abudu** [comment]
ZeaBur貌似还处于Beta阶段，我不太有意愿对其专门适配。
如果你愿意研究的话，可以贡献文档 Qexo/wiki

**Reply by UangSC** [reply]
好吧~
